src/analysis/Analyses.ts

```diff
@@ -373,7 +373,7 @@ const AnalysisCard = ({
               },
               [icon('lock')]
             ),
-          h(TooltipTrigger, { content: Utils.makeCompleteDate(lastModified) }, [
```

Comment:
not blocking for this PR, but this is why I encourage us to stop using import * in normal module files.  This PR would have had cleaner diffs if import { formatDateTime, otherThings } from 'path/utils.ts" was being changed to remove formatDateTime from utils import.